pom.xml

```diff
@@ -77,6 +77,8 @@
     <project.findbugs.version>3.0.2</project.findbugs.version>
     <deploy.autorelease>false</deploy.autorelease>
     <project.error-prone.version>2.35.1</project.error-prone.version>
+    <project.protobuf.version>4.28.3</project.protobuf.version>
```

Comment:
The Java SDK protos are generated with Protoc 25.5. Libraries-Bom will set the Protobuf-Java runtime version to [4.28.3](https://github.com/googleapis/java-cloud-bom/blob/18b9062657f953eb44d2e9cf4945c6bfbf1574aa/libraries-bom/pom.xml#L62-L71). I think this should be 3.25.5 since this is part of the BOM.

Reply:
Actually downgrading it to 3.25.5 causes `RequireUpperBoundDeps` issue since CEL depends on 4.28.2:

```
[ERROR] Rule 2: org.apache.maven.enforcer.rules.dependency.RequireUpperBoundDeps failed with message:
[ERROR] Failed while enforcing RequireUpperBoundDeps. The error(s) are [
[ERROR] Require upper bound dependencies error for com.google.protobuf:protobuf-java:3.25.5 paths to dependency are:
[ERROR] +-com.google.auth:cab-token-generator:1.29.1-SNAPSHOT
[ERROR]   +-com.google.protobuf:protobuf-java:3.25.5
[ERROR] and
[ERROR] +-com.google.auth:cab-token-generator:1.29.1-SNAPSHOT
[ERROR]   +-dev.cel:cel:0.8.0
[ERROR]     +-com.google.protobuf:protobuf-java:3.25.5 (managed) <-- com.google.protobuf:protobuf-java:4.28.2
[ERROR] and
[ERROR] +-com.google.auth:cab-token-generator:1.29.1-SNAPSHOT
[ERROR]   +-dev.cel:cel:0.8.0
[ERROR]     +-com.google.protobuf:protobuf-java-util:4.28.2
[ERROR]       +-com.google.protobuf:protobuf-java:3.25.5 (managed) <-- com.google.protobuf:protobuf-java:4.28.2
[ERROR] ]
```